.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  env: {
+    es6: true,
+    node: true
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:import/recommended',
+    'plugin:mocha/recommended',
+    'plugin:node/recommended',
+    'plugin:promise/recommended',
+    'prettier-standard'
+  ],
+  parserOptions: {
+    ecmaVersion: 2017
+  },
+  plugins: [
+    'import',
+    'mocha',
+    'node',
+    'prettier',
+    'promise', // needed for standard
+    'standard'
+  ],
+  rules: {
```

Comment:
I'd really like to add `'comma-dangle': ['warn', 'always-multiline'],`, but `standard` doesn't seem to agree.